libvast/src/system/accountant.cpp

```diff
@@ -179,9 +241,8 @@ accountant_type::behavior_type accountant(accountant_actor* self,
             VAST_TRACE(self, "received a report from", self->current_sender());
             time ts = std::chrono::system_clock::now();
             for (const auto& [key, value] : r) {
-              auto f = [&, key = key](const auto& x) {
-                record(self, key, x, ts);
-              };
+              auto f
+                = [&, key = key](const auto& x) { record(self, key, x, ts); };
               caf::visit(f, value);
             }
           },
```

Comment:
This lambda sequence looks like it could benefit from re-formatting. However, you did the right thing by not doing it together with this commit, which makes the changes stand out clearly.

Reply:
`clang-format` keeps this entire behavior as it is...

Reply:
Okay.